algolia_settings.json

```diff
@@ -49,8 +49,8 @@
    "geo",
    "words",
    "filters",
-   "proximity",
    "asc(deprecated)",
+   "proximity",
```

Comment:
This change makes it less likely that a deprecated guide will appear above a non-deprecated guide